libraries/AP_NavEKF3/AP_NavEKF3_OptFlowFusion.cpp

```diff
@@ -265,12 +264,13 @@ void NavEKF3_core::EstimateTerrainOffset(const of_elements &ofDataDelayed)
  * The script file used to generate these and other equations in this filter can be found here:
  * https://github.com/PX4/ecl/blob/master/matlab/scripts/Inertial%20Nav%20EKF/GenerateNavFilterEquations.m
  * Requires a valid terrain height estimate.
+ *
+ * really_fuse should be true to actually fuse into the main filter, false to only calculate variances
 */
-void NavEKF3_core::FuseOptFlow(const of_elements &ofDataDelayed)
+void NavEKF3_core::FuseOptFlow(const of_elements &ofDataDelayed, bool really_fuse)
```

Comment:
I would have it fuse by default and have it inhibit fusion by exception

..., bool inhibit_fusion)

Reply:
ok, renamed really_fuse to inhibit_fusion and reversed the set and usage of the argument.